package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "1.2.7",
+  "version": "1.2.8",
```

Comment:
`feat` means a minor version bump, yeah?
```suggestion
  "version": "1.3.0",
```

Reply:
Yes, if we are being strict about conventional commits, but I don't think we really do that right now. I'll follow the rule in the future :)